### ISSUE TITLE
Error: Can not place order (Grid)

### ISSUE THREAD

**Issue by arthurwolf** [c9s/bbgo#434]
![Screenshot from 2022-01-13 12-20-56](https://user-images.githubusercontent.com/108821/149341609-eb86ea50-9e68-43fb-ae25-b559cbb18288.png)
![Screenshot from 2022-01-13 12-19-05](https://user-images.githubusercontent.com/108821/149341612-871fd8af-f55c-430f-b29f-224d3e501ee7.png)

**Comment by zenixls2** [comment]
@arthurwolf 
this issue is also addressed in pr #446   
but the upper price and lower price settings seem incorrect.  
use the following yaml instead:
```yaml
---
sessions:
  binance:
    exchange: binance
    envVarPrefix: binance
backtest:
  startTime: "2021-06-01"
  endTime: "2021-06-02"
  symbols:
  - DNTBTC
  account:
    makerCommission: 15 # TODO: Set the right values for binance, figure out what the number means
    takerCommission: 15
    balances:
      DNT: 0.0
      BTC: 10000

exchangeStrategies:
- on: binance
  grid:
    symbol: DNTBTC
    amount: 66.66666666666667140362
    gridNumber: 150.00000000000000000000
    profitSpread: 1.67039999999999788649
    upperPrice: 0.000004986
    lowerPrice: 0.000004846
    long: true
```
with the given gridNumber and profitSpread, gridSpread will become zero and cause infinite loop.  
add additional check inside grid strategy.  
also according to binance application, dnt/btc should be far less than 1. Use 0.000004986 instead.

**Comment by c9s** [comment]
since we have changed a lot of things, I will close this one, please feel to re-open this if this issue still exist.
